### DISCUSSION TITLE
Imperial conversion in fractions (Not just decimals)

### DISCUSSION THREAD

**Discussion by trymeouteh** [ferraridamiano/ConverterNOW#147] [Ideas]
When converting from metric to imperial in many units such as length, you get a decimal result for the imperial unit. When you use length in Imperial, you do not go by the decimal by in fractions such as 1 3/4 inches for example instead of 1.75 inches.

I not against removing decimal results for imperial conversions but to also include fractional results which can be done in a few ways.

Lets say the imperial decimal result is 1.8 inches. This is between 1 3/4 and 1 13/16 inches and the result can be displayed as between 1 3/4 - 1 13/16.

It can also be displayed as ~1 13/16 since it is approx 1 13/16.

A setting could be added on how much to break down a fractional result and I think 16th of a unit should he the default setting since its not too common to have to break down an imperial unit more than 16 fractional units.

**Comment by ferraridamiano** [comment]
I created a dedicated issue (#148) about this. Let's discuss it there
